### PR TITLE
List: fix highlighting of items and selectAll checkbox in material theme (T1156837)

### PR DESCRIPTION
**Pull request by Krijovnick**

### CODE REVIEW

File: testing/testcafe/tests/editors/list/focus.ts [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    .ok();
+
+  await testScreenshot(t, takeScreenshot, 'List selectAll checkbox should be checked after enter click.png', { element: '#container' });
+
+  await t
+    .expect(compareResults.isValid())
+    .ok(compareResults.errorMessages());
+}).before(async () => createList('all'));
+
+['single', 'multiple'].forEach((selectionMode) => {
```
Comment (EugeniyKiyashko) [inline testing/testcafe/tests/editors/list/focus.ts:49]: We may use single `forEach` for both tests
`Should apply styles on list item after tab button click, ${selectionMode} mode` & `Should apply styles on list item after enter button click on it, ${selectionMode} mode`

File: testing/testcafe/tests/editors/list/focus.ts [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+fixture`List`
+  .page(url(__dirname, '../../container.html'));
+
+const createList = (selectionMode) => createWidget('dxList', {
+  items: ['item1', 'item2', 'item3'],
+  showSelectionControls: true,
+  selectionMode,
+});
+
+test('Should apply styles on selectAll checkbox after tab button click', async (t) => {
```
Comment (ksercs) [inline testing/testcafe/tests/editors/list/focus.ts]: let's change "click" to "press" everywhere, it's a little confusing
```suggestion
test('Should apply styles on selectAll checkbox after tab button press', async (t) => {
```

**Review by ksercs — DISMISSED** [review]

File: testing/testcafe/tests/editors/list/focus.ts (hunk 1 above)
Comment (Krijovnick) [reply testing/testcafe/tests/editors/list/focus.ts:49]: fixed

File: testing/testcafe/tests/editors/list/focus.ts (hunk 2 above)
Comment (Krijovnick) [reply testing/testcafe/tests/editors/list/focus.ts]: fixed

**Review by ksercs — APPROVED** [review]

**Review by EugeniyKiyashko — APPROVED** [review]

**Review by Shuvaevmedia — APPROVED** [review]

**Pull request merged**